src/player.h

```diff
@@ -858,7 +858,7 @@ class player : public Character, public JsonSerializer, public JsonDeserializer
             } else if( has_trait( trait_id( "BADBACK" ) ) ) {
                 str /= 1.35;
             }
-            return get_str() >= obj.lift_strength();
+            return str >= obj.lift_strength();
```

Comment:
This appeared to be a bug that ended up making Bad/Strong back not actually influence the lift strengths of the character like they should have.